src/optimizer/join_order/relation_manager.cpp

```diff
@@ -111,9 +111,11 @@ bool ExpressionContainsColumnRef(Expression &expression) {
 	if (expression.type == ExpressionType::BOUND_COLUMN_REF) {
 		// Here you have a filter on a single column in a table. Return a binding for the column
 		// being filtered on so the filter estimator knows what HLL count to pull
+#ifdef DEBUG
```

Comment:
was there a test that was failing with this in? I can take a look if so

Reply:
This was giving a warning when built in release mode, because the colref wasn't used (assertions optimized out in release).